src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReaderService {
```

Comment:
I think better to have one **FileServise** with read and write methods.
And about name (ReaderService) - **FileReader** or **ReadService** are more correct